app-server/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.8
+
+WORKDIR /app
+COPY man.png /app
```

Comment:
why do you need this image?

Reply:
just to use COPY command and being able to visualize something when server is up)